core/amber/src/main/scala/edu/uci/ics/texera/web/ObserverManager.scala

```diff
@@ -0,0 +1,21 @@
+package edu.uci.ics.texera.web
+
+import rx.lang.scala.Observer
```

Comment:
We seems to have chosen rxscala as our observable library, I didn't recall that I was involved in the discussion. However, rxscala is no longer maintained, should we still use it?
https://github.com/ReactiveX/RxScala

Reply:
Rxscala is internally using RxJava. We can switch to RxJava or Akka Streams if needed.